int8 left_right
int8 front_back
int8 yawl_yawr
int8 up_down